doc/openapi.yaml

```
@@ -682,8 +743,26 @@ paths:
                     "constant": 2
                   },
                   "maxTxsPerBlock": 100,
+                  "rewardParams": {
+                      "compoundingRatio": {
+                          "denominator": 1024,
+                          "numerator": 1
+                      },
+                      "compoundingType": "Linear",
+                      "epochRate": 4294967295,
```

this epoch rate is not likely on normal operation of the blockchain.

```suggestion
                      "epochRate": 100,
```